### PR TITLE
[EDU-2136] Add watermark parameters and example

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Added new supported parameters
- Added example with Azion logo

### Additional links

[EDU-2136]: https://aziontech.atlassian.net/browse/EDU-2136?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/products/edge-application/image-processor.mdx [hunk 1]
```diff
@@ -151,14 +151,24 @@ If you insert a resolution larger than the original, the image will be displayed
...
-- `Y` represents the vertical position for the watermark's insertion. Positive numbers represent the offset, in pixels, from top to bottom, while negative numbers represent the offset from bottom to top. You can use the value *center*, to center the watermark vertically, or the value *repeat*, to fill the image vertically with repetitions of the watermark.
-- `Alpha` is the transparency of the watermark. It must be a number between 0 (completely opaque) and 100 (completely transparent).
-
-For example, the query `?ims=filters:watermark(http://yourdomain.com/watermark-image.png,-25,-10,50)` will apply the watermark 25 pixels left of center and 10 pixels below center, at 50% transparency.
+Add watermarks to your images by appending the query `?ims=filters:watermark(WatermarkURL,X,Y,Alpha,WidthRatio,HeightRatio)`, where:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/products/edge-application/image-processor.mdx:154]:
```suggestion
You can add watermarks to your images by appending the query `?ims=filters:watermark(WatermarkURL,X,Y,Alpha,WidthRatio,HeightRatio)`, where:
```

File: src/content/docs/en/pages/products/edge-application/image-processor.mdx [hunk 2]
```diff
@@ -151,14 +151,24 @@ If you insert a resolution larger than the original, the image will be displayed
...
+
+- `WatermarkURL` is the URL of the image that you wish to insert as a watermark. If the URL contains parentheses, they must be URL encoded as `%28` for `(` and `%29` for `)`.
+- `X` represents the horizontal position for the watermark's insertion. Positive numbers represent the offset, in pixels, from left to right, while negative numbers represent offset from right to left. 
+  - Enter the value `center`, to center the watermark horizontally. 
+  - Enter the value `repeat`, to fill the image with horizontal repetitions of the watermark.
+  - Enter a positive or negative number followed by a 'p' (for example, `20p`) to calculate the horizontal position as a percentage of the image width.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/products/edge-application/image-processor.mdx:160]:
```suggestion
  - Enter a positive or negative number followed by a `p` (for example, `20p`) to calculate the horizontal position as a percentage of the image width.
```

File: src/content/docs/en/pages/products/edge-application/image-processor.mdx [hunk 3]
```diff
@@ -151,14 +151,24 @@ If you insert a resolution larger than the original, the image will be displayed
...
+- `Y` represents the vertical position for the watermark's insertion. Positive numbers represent the offset, in pixels, from top to bottom, while negative numbers represent the offset from bottom to top.  
+  - Enter the value `center`, to center the watermark vertically. 
+  - Enter the value `repeat`, to fill the image with vertical repetitions of the watermark.
+  - Enter a positive or negative number followed by a 'p' (for example, `20p`) to calculate the horizontal position as a percentage of the image width.
+- `Alpha` sets the transparency of the watermark. It must be a number between `0` (completely opaque) and `100` (completely transparent).
+- `WidthRatio` is an optional parameter that sets the watermark width as a percentage relative to the image. If you don't specify a value, it will default to `none`.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/products/edge-application/image-processor.mdx:166]:
```suggestion
- `WidthRatio` is an optional parameter that sets the watermark width as a percentage relative to the image. If you don't specify a value, it'll default to `none`.
```

File: src/content/docs/en/pages/products/edge-application/image-processor.mdx [hunk 4]
```diff
@@ -151,14 +151,24 @@ If you insert a resolution larger than the original, the image will be displayed
...
+- `HeightRatio` is an optional parameter that sets the watermark height as a percentage relative to the image. If you don't specify a value, it will default to `none`.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/products/edge-application/image-processor.mdx:167]:
```suggestion
- `WidthRatio` is an optional parameter that sets the watermark width as a percentage relative to the image. If you don't specify a value, it'll default to `none`.
- `HeightRatio` is an optional parameter that sets the watermark height as a percentage relative to the image. If you don't specify a value, it'll default to `none`.
```

Comment (MarianaAguilera) [inline src/content/docs/en/pages/products/edge-application/image-processor.mdx:167]:
```suggestion
- `HeightRatio` is an optional parameter that sets the watermark height as a percentage relative to the image. If you don't specify a value, it'll default to `none`.
```

File: src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx [hunk 5]
```diff
@@ -151,14 +151,24 @@ Se você inserir uma resolução maior que a original, a imagem será exibida co
...
-- `Y` representa a posição vertical para a inserção da marca d'água. Os números positivos representam o deslocamento, em pixels, de cima para baixo, enquanto os números negativos representam o deslocamento de baixo para cima. Você pode usar o valor *center*, para centralizar a marca d'água verticalmente, ou o valor *repeat*, para preencher a imagem verticalmente com repetições da marca d'água.
-- `Alpha` é a transparência da marca d'água. Deve ser um número entre 0 (completamente opaco) e 100 (completamente transparente).
-
-Por exemplo, a query `?ims=filters:watermark(http://seudominio.com/watermark-image.png,-25,-10,50)` aplicará uma marca d'água 25 pixels à esquerda do centro e 10 pixels abaixo do centro, com 50% de transparência.
+Adicione marcas d'água às suas imagens acrescentando a query `?ims=filters:watermark(WatermarkURL,X,Y,Alpha,WidthRatio,HeightRatio)`, onde:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx:154]:
```suggestion
Você pode adicionar marcas d'água às suas imagens acrescentando a query `?ims=filters:watermark(WatermarkURL,X,Y,Alpha,WidthRatio,HeightRatio)`, onde:
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx [hunk 6]
```diff
@@ -151,14 +151,24 @@ Se você inserir uma resolução maior que a original, a imagem será exibida co
...
+
+- `WatermarkURL` é a URL da imagem que você deseja inserir como marca d'água. Se a URL contém parênteses, eles devem ser codificados como `%28` para `(` e `%29` para `)`.
+- `X` representa a posição horizontal para a inserção da marca d'água. Números positivos representam o deslocamento, em pixels, da esquerda para a direita, enquanto números negativos representam o deslocamento da direita para a esquerda.
+  - Insira o valor `center`, para centralizar a marca d'água horizontalmente.
+  - Insira o valor `repeat`, para preencher a imagem com repetições horizontais da marca d'água.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx:159]:
```suggestion
  - Insira o valor `center`,para centralizar a marca d'água horizontalmente.
  - Insira o valor `repeat` para preencher a imagem com repetições horizontais da marca d'água.
```

File: src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx [hunk 7]
```diff
@@ -151,14 +151,24 @@ Se você inserir uma resolução maior que a original, a imagem será exibida co
...
+  - Insira um número positivo ou negativo seguido de um 'p' (por exemplo, `20p`) para calcular a posição horizontal como uma porcentagem da largura da imagem.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx:160]:
```suggestion
  - Insira um número positivo ou negativo seguido de um `p` (por exemplo, `20p`) para calcular a posição horizontal como uma porcentagem da largura da imagem.
```

File: src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx [hunk 8]
```diff
@@ -151,14 +151,24 @@ Se você inserir uma resolução maior que a original, a imagem será exibida co
...
+- `Y` representa a posição vertical para a inserção da marca d'água. Números positivos representam o deslocamento, em pixels, de cima para baixo, enquanto números negativos representam o deslocamento de baixo para cima.
+  - Insira o valor `center`, para centralizar a marca d'água verticalmente.
+  - Insira o valor `repeat`, para preencher a imagem com repetições verticais da marca d'água.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx:163]:
```suggestion
  - Insira o valor `center` para centralizar a marca d'água verticalmente.
  - Insira o valor `repeat` para preencher a imagem com repetições verticais da marca d'água.
```

File: src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx [hunk 9]
```diff
@@ -151,14 +151,24 @@ Se você inserir uma resolução maior que a original, a imagem será exibida co
...
+  - Insira um número positivo ou negativo seguido por um 'p' (por exemplo, `20p`) para calcular a posição vertical como uma porcentagem da largura da imagem.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx:164]:
```suggestion
  - Insira um número positivo ou negativo seguido por um `p` (por exemplo, `20p`) para calcular a posição vertical como uma porcentagem da largura da imagem.
```

**Review by hannah-sk — CHANGES_REQUESTED** [review]
some suggestions + style corrections

File: src/content/docs/en/pages/products/edge-application/image-processor.mdx [hunk 10]
```diff
@@ -151,14 +151,24 @@ If you insert a resolution larger than the original, the image will be displayed
...
+
+- `WatermarkURL` is the URL of the image that you wish to insert as a watermark. If the URL contains parentheses, they must be URL encoded as `%28` for `(` and `%29` for `)`.
+- `X` represents the horizontal position for the watermark's insertion. Positive numbers represent the offset, in pixels, from left to right, while negative numbers represent offset from right to left. 
+  - Enter the value `center`, to center the watermark horizontally. 
+  - Enter the value `repeat`, to fill the image with horizontal repetitions of the watermark.
+  - Enter a positive or negative number followed by a 'p' (for example, `20p`) to calculate the horizontal position as a percentage of the image width.
+- `Y` represents the vertical position for the watermark's insertion. Positive numbers represent the offset, in pixels, from top to bottom, while negative numbers represent the offset from bottom to top.  
+  - Enter the value `center`, to center the watermark vertically. 
+  - Enter the value `repeat`, to fill the image with vertical repetitions of the watermark.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/products/edge-application/image-processor.mdx:163]:
```suggestion
  - Enter the value `center` to center the watermark vertically. 
  - Enter the value `repeat` to fill the image with vertical repetitions of the watermark.
```

File: src/content/docs/en/pages/products/edge-application/image-processor.mdx [hunk 11]
```diff
@@ -151,14 +151,24 @@ If you insert a resolution larger than the original, the image will be displayed
...
-- `Alpha` is the transparency of the watermark. It must be a number between 0 (completely opaque) and 100 (completely transparent).
-
-For example, the query `?ims=filters:watermark(http://yourdomain.com/watermark-image.png,-25,-10,50)` will apply the watermark 25 pixels left of center and 10 pixels below center, at 50% transparency.
+Add watermarks to your images by appending the query `?ims=filters:watermark(WatermarkURL,X,Y,Alpha,WidthRatio,HeightRatio)`, where:
+
+- `WatermarkURL` is the URL of the image that you wish to insert as a watermark. If the URL contains parentheses, they must be URL encoded as `%28` for `(` and `%29` for `)`.
+- `X` represents the horizontal position for the watermark's insertion. Positive numbers represent the offset, in pixels, from left to right, while negative numbers represent offset from right to left. 
+  - Enter the value `center`, to center the watermark horizontally. 
+  - Enter the value `repeat`, to fill the image with horizontal repetitions of the watermark.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/products/edge-application/image-processor.mdx:159]:
```suggestion
  - Enter the value `center`, to center the watermark horizontally. 
  - Enter the value `repeat` to fill the image with horizontal repetitions of the watermark.
```

File: src/content/docs/en/pages/products/edge-application/image-processor.mdx (hunk 1 above)
Comment (bru-andrade) [reply src/content/docs/en/pages/products/edge-application/image-processor.mdx:154]: Estou deixando como padrão o imperativo mesmo, como no style guide que a Mari

File: src/content/docs/en/pages/products/edge-application/image-processor.mdx [hunk 12]
```diff
@@ -151,14 +151,24 @@ If you insert a resolution larger than the original, the image will be displayed
...
+- `Alpha` sets the transparency of the watermark. It must be a number between `0` (completely opaque) and `100` (completely transparent).
+- `WidthRatio` is an optional parameter that sets the watermark width as a percentage relative to the image. If you don't specify a value, it'll default to `none`.
+- `HeightRatio` is an optional parameter that sets the watermark height as a percentage relative to the image. If you don't specify a value, it'll default to `none`.
+
+For example, the query `?ims=filters:watermark(http://yourdomain.com/watermark-image.png,repeat,20p,50,10,none)` will repeat the watermark horizontally, at 20% image height, at 50% transparency, resized to 10% of the image width.
+
+<p align="center">[![Image_Processor_1](/assets/docs/images/image-processor/Image_Processor_2.png?ims=filters:watermark(https://raw.githubusercontent.com/aziontech/docs/main/public/assets/docs/images/azion-logo.png,repeat,20p,50,10,none))](/assets/docs/images/image-processor/Image_Processor_2.png?ims=filters:watermark(https://raw.githubusercontent.com/aziontech/docs/main/public/assets/docs/images/azion-logo.png,repeat,20p,50,10,none))</p>
```
Comment (peeetry) [inline src/content/docs/en/pages/products/edge-application/image-processor.mdx:171]: hmm

File: src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx [hunk 13]
```diff
@@ -151,14 +151,24 @@ Se você inserir uma resolução maior que a original, a imagem será exibida co
...
+- `WidthRatio` é um parâmetro opcional que define a largura da marca d'água como uma porcentagem em relação à imagem. Se você não especificar um valor, ele será definido como `none`.
+- `HeightRatio` é um parâmetro opcional que define a altura da marca d'água como uma porcentagem em relação à imagem. Se você não especificar um valor, ele será definido como `none`.
+
+Por exemplo, a query `?ims=filters:watermark(http://seudominio.com/imagens-marca-dagua.png,repeat,20p,50,10,none)` repetirá a marca d'água horizontalmente, a 20% da altura da imagem, com 50% de transparência, redimensionada para 10% da largura da imagem.
+
+<p align="center">[![Image_Processor_1](/assets/docs/images/image-processor/Image_Processor_2.png?ims=filters:watermark(https://raw.githubusercontent.com/aziontech/docs/main/public/assets/docs/images/azion-logo.png,repeat,20p,50,10,none))](/assets/docs/images/image-processor/Image_Processor_2.png?ims=filters:watermark(https://raw.githubusercontent.com/aziontech/docs/main/public/assets/docs/images/azion-logo.png,repeat,20p,50,10,none))</p>
```
Comment (peeetry) [inline src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx:171]: hmm

File: src/content/docs/en/pages/products/edge-application/image-processor.mdx [hunk 14]
```diff
@@ -151,14 +151,24 @@ If you insert a resolution larger than the original, the image will be displayed
...
-- `Alpha` is the transparency of the watermark. It must be a number between 0 (completely opaque) and 100 (completely transparent).
-
-For example, the query `?ims=filters:watermark(http://yourdomain.com/watermark-image.png,-25,-10,50)` will apply the watermark 25 pixels left of center and 10 pixels below center, at 50% transparency.
+Add watermarks to your images by appending the query `?ims=filters:watermark(WatermarkURL,X,Y,Alpha,WidthRatio,HeightRatio)`, where:
+
+- `WatermarkURL` is the URL of the image that you wish to insert as a watermark. If the URL contains parentheses, they must be URL encoded as `%28` for `(` and `%29` for `)`.
+- `X` represents the horizontal position for the watermark's insertion. Positive numbers represent the offset, in pixels, from left to right, while negative numbers represent offset from right to left. 
+  - Enter the value `center` to center the watermark horizontally. 
+  - Enter the value `repeat`, to fill the image with horizontal repetitions of the watermark.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/products/edge-application/image-processor.mdx:159]:
```suggestion
  - Enter the value `repeat` to fill the image with horizontal repetitions of the watermark.
```

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx (hunk 13 above)
Comment (bru-andrade) [reply src/content/docs/pt-br/pages/produtos/edge-application/image-processor/index.mdx:171]: https://www.azion.com/assets/docs/images/image-processor/Image_Processor_2.png?ims=filters:watermark(https://raw.githubusercontent.com/aziontech/docs/main/public/assets/docs/images/azion-logo.png,repeat,20p,50,10,none)

**Pull request merged**